src/olympia/zadmin/forms.py

```diff
@@ -31,7 +31,8 @@ class DevMailerForm(happyforms.Form):
     _choices = [('eula',
                  'Developers who have set up EULAs for active add-ons'),
                 ('sdk', 'Developers of active SDK add-ons'),
-                ('all_extensions', 'All extension developers')]
+                ('all_extensions', 'All extension developers'),
+                ('fxa', 'Developers who have to migrate to FxA')]
```

Comment:
How about: `Developers who still have to migrate to FxA`. Since I'm one pushing the button in a week or so's time I think I'll be ok.